contrib/mod_quotatab_ldap.c

```diff
@@ -65,17 +65,19 @@ static unsigned char ldaptab_lookup(quota_table_t *ldaptab, void *ptr,
   ldap_cmd = pr_cmd_alloc(tmp_pool, 1, name);
   ldap_res = pr_module_call(ldap_cmdtab->m, ldap_cmdtab->handler, ldap_cmd);
 
-  destroy_pool(tmp_pool);
-
```

Comment:
This _might_ be destroying this temporary pool too soon; the returned `ldap_res` structure may have used this pool, and we reference that structure _after_ the temporary pool is destroyed.  So let's destroy the pool only after we've done everything else.